samples/apps/copilot-chat-app/webapi/SkServiceConstants.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace SemanticKernel.Service;
 
-internal static class CopilotChatApiConstants
+internal static class SkServiceConstants
```

Comment:
[nit] Consider removing redundant prefixes. Maybe we just call this class "Constants" since it's already in the SemanticKernel.Service namespace.

Reply:
Done